pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -2283,6 +2358,30 @@ def btn_stack_prev_clicked(self, event):
             if n_current > 0:
                 self.switch_to_different_stack(self.labels[n_current - 1])
 
+        def btn_save_spectrum_clicked(self, event):
+            # Create file name for the CSV file
+            fln = "spectrum"
+            fln += f"_{self.label_selected}"
```

Comment:
Can it be done in one line?
```suggestion
            fln = f"spectrum_{self.label_selected}"
```